.github/workflows/build-hub.yaml

```diff
@@ -52,34 +52,38 @@ jobs:
           check-latest: true
           cache-dependency-path: |
             go.sum
+      - name: Install pnpm
+        uses: pnpm/action-setup@41ff72655975bd51cab0327fa583b6e92b6d3061 # v4.2.0
+        with:
+          version: 10
       - name: Setup node.js
         uses: actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903 # v6.0.0
         with:
           node-version: '24'
-          cache: npm
+          cache: pnpm
           registry-url: 'https://registry.npmjs.org'
       - name: Generate commit hash
         id: hash
         run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
 
-      - name: Install frontend dependencies with npm
-        run: npm ci
+      - name: Install frontend dependencies with pnpm
+        run: pnpm install --frozen-lockfile
 
       - name: Lint with prettier
-        run: npm run lint
+        run: pnpm run lint
       - name: Lint with golangci-lint
         uses: golangci/golangci-lint-action@0a35821d5c230e903fcfe077583637dea1b27b47 # v9.0.0
         with:
           version: v2.5.0
 
       - name: Build frontend
         if: ${{ !startsWith(github.ref, 'refs/tags/') }}
-        run: npm run build:prod
+        run: pnpm run build:prod
       - name: Build frontend and upload source maps (tag only)
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         run: |
-          npm run build:prod -- --source-map=true || exit 1
-          npm run sentry-upload || exit 1
+          pnpm run build:prod -- --source-map=true || exit 1
```

Comment:
for pnpm it is not possible to use -- multiple times:

> @glasskube/distr@1.15.2 build
> ng build --configuration=production -- --source-map=true

```
Option '--' has been specified multiple times. The value '--source-map=true' will be used.
An unhandled exception occurred: Schema validation failed with the following errors:
  Data path "" must NOT have additional properties().                                                                               
See "/tmp/ng-2FaJQN/angular-errors.log" for further details.                                                                        
 ELIFECYCLE  Command failed.
```

Reply:
good catch, thanks